refinery/data_set_manager/views.py

```diff
@@ -497,29 +515,55 @@ def post(self, request, *args, **kwargs):
         if not request.is_ajax() or not request.body:
             return HttpResponseBadRequest()
 
-        file_data = json.loads(request.body)
         try:
-            base_path = file_data["base_path"]
+            file_data = json.loads(request.body)
+        except ValueError:
+            return HttpResponseBadRequest()
+        try:
+            input_file_list = file_data['list']
         except KeyError:
-            base_path = ""
+            return HttpResponseBadRequest()
```

Comment:
`base_path` is optional, so no response necessary. The file list would not be present only if the are no files in the meta data file. I'm not sure how could that happen. If this is of concern, it should be a separate issue.